File: server/src/main/java/com/mthwate/conk/command/CommandStop.java
package com.mthwate.conk.command;

import com.mthwate.conk.ServerApp;

/**
 * @author mthwate
 */
@Exec
public class CommandStop implements Command {

	@Override
	public String getName() {
		return "stop";
	}

	@Override
	public void run(ServerApp app, String[] args) {
		app.stop();
	}

}

File: server/src/main/kotlin/com/mthwate/conk/command/CommandStop.kt
package com.mthwate.conk.command

import com.mthwate.conk.ServerApp

/**
 * @author mthwate
 */
@Exec
class CommandStop : Command {

	override val name = "stop"

	override fun run(app: ServerApp, args: Array<String>) = app.stop()

}
